adapter/k8s.go

```diff
@@ -56,6 +56,7 @@ func (h *Adapter) CreateNamespace(isDelete bool, namespace string) error {
 	return nil
 }
 
+// GetServicePorts returns the node port(s) for a specific service in the namespace given.
```

Comment:
This is great. Do we have a function to "GetServiceHostname"? If not, we need such a function.

Reply:
why ServiceHostname @leecalcote ?

Reply:
With GetServicePorts, you can inform the user of what port a service has been deployed to, but without knowing the `exernal-ip` (hostname) or the `cluster-ip`, you can't connect them directly to the service.